pkg/pillar/cas/cas.go

```diff
@@ -105,6 +105,9 @@ type CAS interface {
 	// The rootPath is expected to end in a basename that becomes the snapshotID
 	PrepareContainerRootDir(rootPath, reference, rootBlobSha string) error
 
+	// UnmountContainerRootDir unmounts container's rootPath.
+	UnmountContainerRootDir(rootPath string) error
```

Comment:
I really want this to be parallel, so `UnprepareContainerRootDir()`, but that is just so awful, let's leave it as `UnmountContainerRootDir()` 😄